index.js

```diff
@@ -9,12 +9,10 @@ module.exports = {
     NewParty: Deployer.abi.find(({ type, name }) => type === 'event' && name === 'NewParty'),
     /* when someone registers for a party */
     Register: Conference.abi.find(({ type, name }) => type === 'event' && name === 'RegisterEvent'),
-    /* when someone is marked as having attended a party */
-    Attend: Conference.abi.find(({ type, name }) => type === 'event' && name === 'AttendEvent'),
-    /* when someone withdraws their payout */
-    Withdraw: Conference.abi.find(({ type, name }) => type === 'event' && name === 'WithdrawEvent'),
     /* when party attendance is finalized and payout is enabled */
     Finalize: Conference.abi.find(({ type, name }) => type === 'event' && name === 'FinalizeEvent'),
+    /* when someone withdraws their payout */
+    Withdraw: Conference.abi.find(({ type, name }) => type === 'event' && name === 'WithdrawEvent'),
     /* when the party has ended */
     EndParty: Conference.abi.find(({ type, name }) => type === 'event' && name === 'PaybackEvent'),
     /* when the party gets cancelled */
```

Comment:
`PaybackEvent` is redundant to `FinalizeEvent` in terms of envoking timing. If we want to pass the payout amount we can send it as a second argument of `FinalizeEvent`.

Reply:
Agreed